app/javascript/src/Dashboard/tabs-widget.jsx

```diff
@@ -0,0 +1,19 @@
+const CURRENT_TAB_CLASS = 'current-tab'
+
+export function initialize () {
+  const tabProducts = document.querySelector('label[for="tab-products"]')
+  const tabBackends = document.querySelector('label[for="tab-backends"]')
+
+  const inputProducts = document.querySelector('input#tab-products')
+  const inputBackends = document.querySelector('input#tab-backends')
+
+  inputProducts.addEventListener('change', () => {
```

Comment:
Maybe avoiding repetition with something like..
```
[inputProducts, inputBackends].forEach(
	input => input.addEventListener('change', () => [tabProducts, tabBackends].forEach(
	tab => tab.classList.toggle(CURRENT_TAB_CLASS)
	))
)
```

Reply:
I wanted to avoid using `toggle` because if somehow the same `change` event is triggered twice (it is possible programmatically), the tabs will look wrong. Maybe I shouldn't be concerned about it?

Reply:
I wouldn't be too concerned about it, still this is not something important to change.. I'd prioritize the migration of the js in the view to the pack.